docs/sources/administration/configuration.md

```diff
@@ -623,6 +623,13 @@ options are `Admin` and `Editor`. e.g.:
 
 `auto_assign_org_role = Viewer`
 
+### skip_social_org_role_assign
+
+Skip forced assignment of OrgID `1` or `auto_assign_org_id` for social logins.
+This allows to manually distribute users with social login to multiple organizations.
+Otherwise, the users' organization would be reset on every new login e.g. via AzureAD.
+Default is `false`.
```

Comment:
```suggestion
Skip forced assignment of OrgID `1` or `auto_assign_org_id` for social logins. Default is `false`.
Use this setting to distribute users with a social login to multiple organizations.
Otherwise, the users' organization would get reset on every new login, for example, via AzureAD.
```

Reply:
thank you very much for the feedback!
I replaced my original draft with your suggestion :blush: